Automattic/wp-calypso
# Wrapping Plans copy AB test

#### Changes proposed in this Pull Request

* Wraps up https://github.com/Automattic/wp-calypso/pull/39729

More information here p8Eqe3-144

#### Testing instructions

1. Go through the signup flow at wordpress.com/start
1. Verify that you are always shown the control version of the Plans step (i.e. no testimonials section). You should not be assigned to the `planStepCopyUpdates ` test since we are removing it. 
1. Add any plan to cart and complete payment on the checkout page.
